src/summary.F90

```diff
@@ -534,6 +535,10 @@ subroutine write_tallies(file_id)
     type(RegularMesh), pointer :: m
     type(TallyObject), pointer :: t
 
+    integer                     :: offset   ! distibcell offset
+    character(100), allocatable :: paths(:) ! array of distribcell paths
+    character(100)              :: path    ! temporary distribcell path
```

Comment:
We should use constant parameters for these lengths.  `MAX_LINE_LEN` is probably the most appropriate based on it's name alone.  Naturally, you should also change the `path` variable in `output % get_label`